astroquery/query.py

```diff
@@ -36,12 +36,14 @@ def _replace_none_iterable(iterable):
 
 class AstroQuery(object):
 
-    def __init__(self, method, url, params=None, data=None, headers=None,
+    def __init__(self, method, url,
+                 params=None, data=None, json=None, headers=None,
```

Comment:
keep the order of kwargs and  add the new one as the last one, otherwise it's an API breakage in the case when they were used as positional args. Arguably we need to force these to be kwarg only at a (hopefully near) future refactor, but that's beyond the scope of this PR.

Reply:
Done.  IMHO it should be an error to stuff keyword args using position.  I think PEP 3102 is an attempt to allow enforcement of that.